docs/UPGRADING.rst

```diff
@@ -0,0 +1,20 @@
+Upgrading HaaS
+==============
+
+This file describes the procedure for upgrading HaaS to a new version.
+
+1. Read the release notes, as they may contain version-specific information.
+2. Stop the HaaS services (webserver and HaaS network daemon).
+3. Download and install the new version of HaaS::
+
+     $ git clone https://github.com/cci-moc/haas
+     $ python setup.py install
+
+3. Upgrade the database::
+
+     $ haas-admin db upgrade heads
+
+4. If additional extensions have been added to ``haas.cfg``, re-run ``haas db
+   create``, which will create any tables needed by those extensions.
```

Comment:
re (1), we don't really support that use case. I'll add a note to that effect.

re (3), no, it doesn't.

Good catch on (2); I'll fix.